src/block-components/alignment/frontend-has-polyfill.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * WordPress dependencies
+ */
+import domReady from '@wordpress/dom-ready'
+
+class StackableAlignmentPolyfill {
+	init = () => {
+		const modifyCSS = () => {
+			const containers = document.querySelectorAll( '.stk-container' )
+			containers.forEach( container => {
+				const columnFlex = container.querySelector( '.stk--column-flex' )
```

Comment:
Based on the selector we're trying to mimic, we need this to be for the direct descendent
```suggestion
				const columnFlex = container.querySelector( ':scope > .stk--column-flex' )
```